agent/www/components/addresses/addresses.html

```diff
@@ -38,22 +38,44 @@
         </div>
       </div>
       <div class="list-view-pf-additional-info">
+        <div class="list-view-pf-additional-info-item">
+          <span title="Messages sent to this address in last 5 minutes">
+            <span class="fa fa-sign-in"></span>
+            <strong>{{item.messages_in_delta}}</strong> Incoming Traffic
```

Comment:
I'm not sure about "Incoming Traffic". I think the user will still be confused.   The hover over helps, but the user would need to know to look for this. I think we need to find a way to display a more descriptive field label. Maybe we could use smaller text beneath the value "Message In - Last 5 min" etc.

I think also the change needs to be reflected on the Connections tab too.

Reply:
> I think the connections tab is different. There the count is the actual number of messages on that connection.

I hadn't appreciated that those were absolute values.  Thanks.

Reply:
@gsim - any thoughts on my comments on the label used on the address tab?

Reply:
@k-wall I don't have strong feelings about the text. I think the time period it covers is really a detail. Though useful to be able to find it out easily, in my view it does not need to be part of the main label; perhaps adding an info icon as the target for the mouseover would help inform users of the extra text available.

Apart for the time period, 'message in' was what was there before which I thought was reported as being unclear.